toqito/state_props/is_separable.py

```diff
@@ -61,7 +61,7 @@ def is_separable(
     :param level: The level up to which to search for the symmetric extensions.
     :param tol: Numerical tolerance used.
     :return: :code:`True` if :code:`rho` is separabale and :code:`False` otherwise.
-    """
+    """  # noqa: E501
```

Comment:
Temporarily disabled line `length > 120` for this docstring. Make changes to `math` before marking this PR ready for review.